Presentr/PresentrController.swift

```diff
@@ -45,11 +62,13 @@ class PresentrController: UIPresentationController, UIAdaptivePresentationContro
          backgroundOpacity: Float,
          blurBackground: Bool,
          blurStyle: UIBlurEffectStyle,
+         keyboardTranslationType : KeyboardTranslationType,
```

Comment:
@aasatt Would be nice to keep the code consistent. Notice that the colon doesn't comes after a space, but right after the property name.